tests/backup/backup_test.go

```diff
@@ -4947,6 +5065,77 @@ func UpdateBackup(backupName string, backupuid string, org_id string, cloudCred
 
 }
 
+// CreateBackup creates backup with custom resources
+func CreateBackupWithCustomResourceType(backupName string, clusterName string, bLocation string, bLocationUID string,
+	namespaces []string, labelSelectors map[string]string, orgID string, uid string, preRuleName string,
+	preRuleUid string, postRuleName string, postRuleUid string, resourceType []string ,ctx context.Context) error {
```

Comment:
Minor formatting issue. There should be a space between comma and `ctx`

Reply:
Done